src/renderer/store/tachyon.store.ts

```diff
@@ -23,7 +23,11 @@ async function connect() {
         tachyonStore.error = undefined;
     } catch (err) {
         console.error("Failed to connect to Tachyon server", err);
-        tachyonStore.error = "Error";
+        if (err instanceof Error && err.message.split(": ")[1]) {
```

Comment:
This is nasty parsing. Proper way is to subtype `Error` and pass in custom properties.